shepherd_bms/include/compute.h

```diff
@@ -177,8 +177,19 @@ class ComputeInterface
        * @param current 
        */
       void sendCurrentsStatus(uint16_t discharge, uint16_t charge, uint16_t current);
+   
+        /**
+       * @brief sends cell temperature message
+       * 
+       * 
+       * 
+       * @return Returns a fault if we are not able to send
+      */
+      void sendCellTemp(uint16_t mCellTemp, uint8_t mCellID, uint16_t cellTemp, uint8_t cellID, uint16_t tempAvg);
```

Comment:
just make sure to change the names here the same way u did in compute.cpp and ur good to go